### DISCUSSION TITLE
Problems with the screenshot in the GTS 4 mini simulator.

### DISCUSSION THREAD

**Discussion by SashaCX75** [zepp-health/zeppos-forum#198] [Developer Feedback]
When I test the software in the GTS 4 mini simulator everything is displayed correctly. But if I try to take a screenshot in the simulator, the bottom part comes out empty.

Example how it looks like in simulator.
![2023-06-19_18-55-10](https://github.com/zepp-health/zeppos-forum/assets/12464527/d80fb591-da9f-4f02-93b2-7089ae709b40)

Example of the screenshot, the problem area is marked in red.
![2023-06-19_18-58-32](https://github.com/zepp-health/zeppos-forum/assets/12464527/68e8f56b-5534-4c79-bae7-f06f286104d5)

**Comment by MyoungXUE** [comment]
Hello, it seems that there is a bug in the function of the screenshot. I have reported this problem to relevant colleagues, thank you for your feedback.
